### ISSUE TITLE
Add schema to GET responses in OpenApi definition

### ISSUE THREAD

**Issue by samuba** [begriffs/postgrest#817]
Right now the OpenApi definition does not provide a schema for the GET responses.
Those definitions already exists (they are e.g. used for POST parameters) and it would be nice to also have them included in the GET responses.
This would make the API definition more complete and enable clients, for example swagger-ui, to always show the model or even example results of the endpoint. 
See for example: http://petstore.swagger.io/#!/pet/getPetById 

Currently the OpenApi definition for a GET endpoint response looks like this:
```
"responses": {
    "206": {
        "description": "Partial Content"
     },
     "200": {
         "description": "OK"
     }
 }
```

Including the schema would look like this:
```
"responses": {
    "206": {
        "description": "Partial Content"
     },
     "200": {
         "description": "OK",
         "schema": {
             "$ref": "#/definitions/film"
          }
     }
 }
```

**Comment by mpwang** [comment]
I think the change is necessary, otherwise when using swagger-codegen, the generated library will return none for any get method

**Comment by ruslantalpa** [comment]
fixed by #885
